graphql_compiler/typedefs.py

```diff
@@ -4,6 +4,12 @@
 from graphql import GraphQLList, GraphQLNonNull, GraphQLScalarType
 
 
+try:
+    from typing import TypedDict
+except ImportError:  # TypedDict was only added to typing in Python 3.8
+    from typing_extensions import TypedDict
```

Comment:
Some some reason flake8 isn't recognizing this as being used, even though we import it.

Reply:
The `flake8` and `pylint` import use analyses run on a per-file basis, they don't expect that we're importing this here to be available for importing from other files.